TradeItIosTicketSDK2/Info.plist

```diff
@@ -15,11 +15,11 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>1.1.16</string>
```

Comment:
See the script below - it will update all of these automatically using an apple tool called `agvtool`. Check here for docs: https://developer.apple.com/library/content/qa/qa1827/_index.html